docs/tiered-storage-overview.md

```diff
@@ -83,6 +83,6 @@ Tiered storage works as follows:
 
 5. Moreover, you can trigger the offloading operation manually (via REST API or CLI) or automatically (via CLI).
 
-6. After offloading ledgers to long-term storage, you can still query data in the offloaded ledgers with Pulsar SQL.
+6. After transferring ledgers to long-term storage, the messages within these ledgers remain accessible to Pulsar consumers and readers, ensuring transparency in data retrieval. Additionally, you can query data in the offloaded ledgers using Pulsar SQL..
```

Comment:
We have removed Pulsar SQL from the distribution since 3.2.0 . I wonder if this could be somehow communicated in this sentence or whether we should simply remove referencing Pulsar SQL.

Reply:
Good catch! I removed the sentence